src/main/kotlin/com/cultureamp/eventsourcing/BookmarkStore.kt

```diff
@@ -17,6 +18,13 @@ class RelationalDatabaseBookmarkStore(val db: Database, val table: Bookmarks = B
         Bookmark(bookmarkName, bookmarkVal)
     }
 
+    override fun bookmarksFor(bookmarkNames: Set<String>): Set<Bookmark> = transaction(db) {
```

Comment:
any reason you don't have `bookmarkFor()` call this?

Reply:
Hmm good point - will postgres optimize a single item in-clause to an eq clause? Just wondering if there is any performance consideration but this table will almost always have less than 100 rows anyway so that part is pretty academic. I feel sort of agnostic about it - if you think it's worthwhile I'm happy to change it

Reply:
TBH I don't really know how postgres handles `in ()` in terms of query planning - I expect it's something simple like "just assume there will be 10 parameters" or something. But in this case I can't see it being an issue. There's only 1 index on the table so it's got a choice - use the index (which it can in this case) or scan the whole table. I'm confident it will use the index for both cases. so it will be equivalent

Reply:
done